jbpm-bpmn2/src/main/java/org/jbpm/bpmn2/xml/IntermediateThrowEventHandler.java

```diff
@@ -50,6 +57,7 @@
 	public static final String LINK_NAME = "linkName";
 	public static final String LINK_SOURCE = "source";
 	public static final String LINK_TARGET = "target";
+    private static final String TRANSFORMATION = "Transformation";
```

Comment:
Problem here is that this file is old and is using old formatting. 
Old line uses tabs, new lines use spaces, thats why is indented differently
I can format all lines, but this will make difficult to track changes related with this jira. 
I do not think using old format is a wise choise, so Ill leave up to you, format old code or leave it that way

Reply:
ok, agree - I wasn't aware of this old format. Thanks.